src/test/java/fr/insee/pearljam/integration/surveyunit/SurveyUnitIT.java

```diff
@@ -231,19 +231,20 @@ void testGetSurveyUnitDetail() throws Exception {
 				   },
 				   "communicationTemplates":[
 				      {
-				         "id":1,
+				         "meshuggahId":"mesh1",
```

Comment:
warning: for retrcompatibility, should expose same old attribute id AND new attribute meshuggahId => same value